src/main.rs

```diff
@@ -23,32 +23,37 @@ struct Cli {
     #[clap(parse(from_str = parse_dependency))]
     dependencies: Vec<Dependency>,
 
+    /// Create a library instead of a binary.
+    #[clap(short, long)]
+    lib: bool,
+
     /// Name of the temporary crate.
     #[clap(long = "name")]
     project_name: Option<String>,
 
-    /// Create a library instead of a binary.
-    #[clap(long)]
-    lib: bool,
+    /// Create a git working tree
+    #[clap(long = "worktree")]
+    worktree_branch: Option<Option<String>>,
 }
 
 #[derive(Debug, PartialEq, Eq)]
 enum Dependency {
     CrateIo(String, Option<String>),
     Repository {
-        name: String,
-        url: String,
         branch: Option<String>,
+        name: String,
         rev: Option<String>,
+        url: String,
```

Comment:
It's okay you can leave this hunk if you want but I must warn you you are creating noise for the reviewer. A PR should try to focus on solving one problem. Deviating a bit with quick fixes is okay but too many quick fixes and your PR becomes unreviewable.

Reply:
Right, that can go on a refactor.